docs/install.md

```diff
@@ -80,38 +98,6 @@ configuration.
 
 ```shell
 dnf -y install podman skopeo
-podman run --rm --privileged --pid=host -v /:/target --security-opt label=type:unconfined_t <yourimage> bootc install to-filesystem --karg=console=ttyS0,115200n8 --replace=alongside /target
+podman run --rm --privileged --pid=host -v /:/target -v /var/lib/containers:/var/lib/containers --security-opt label=type:unconfined_t <yourimage> bootc install to-filesystem --karg=console=ttyS0,115200n8 --replace=alongside /target
```

Comment:
I would tend to believe you, but as it's currently load bearing right now and would need testing to verify this, I'd like to avoid changing that in this PR.  We can do it as a followup?